### DISCUSSION TITLE
Can I join Team

### DISCUSSION THREAD

**Discussion by donboruza** [ROM-builders/temporary#11866] [Q&A]
Hi sir can I join  team ? 
this is the rom that i've managed to make
projectFluid: 
https://cirrus-ci.com/build/6271975411679232

CrDroid:
https://cirrus-ci.com/build/5014002710347776

riceDroid:
https://cirrus-ci.com/build/5014002710347776

spark OS:
https://cirrus-ci.com/build/4677533932191744

riceDroid for new decice rn10pro
https://cirrus-ci.com/build/4749160893644800

Also this is my github account:
https://github.com/donboruza

and thia is my telegram account:
https://t.me/don_boruza

Thank you sir :)

**Comment by Apon77** [comment]
seems like its a fake account.. very new github account. please wait few months.
thank you.

**Reply by donboruza** [reply]
@Apon77 sir now can i join the team?? 
after a few months i'm waiting because before you mentioned "wait for a few months"
I hope it can be approved to join
